client/src/components/ScheduledAppointments/ScheduledAppointmentsTab.js

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from "react";
+import { Grid } from "@material-ui/core";
+import axios from "axios";
+import { parseISO, startOfToday, compareAsc } from "date-fns";
+
+import { useUserData } from "../../providers/Context";
+import IndividualAppointment from "./IndividualAppointment";
+import YourAppointments from "./YourAppointments";
+
+const ScheduledAppointmentsTab = () => {
+  const userData = useUserData();
+  const { _id: userId } = userData;
+
+  const [upcomingAppointments, setupcomingAppointments] = useState([]);
```

Comment:
```suggestion
  const [upcomingAppointments, setUpcomingAppointments] = useState([]);
```
You'll have to change this in other places too